### DISCUSSION TITLE
Non-alpha status for `opentelemetry-kafka-clients-2.6`

### DISCUSSION THREAD

**Discussion by asafch** [open-telemetry/opentelemetry-java-instrumentation#8853] [Q&A] [answered]
I scoured the issues, PRs and discussions in hope of understanding if there's a plan to move `opentelemetry-kafka-clients-2.6` from alpha status to stable, but couldn't find anything for that affect. Am I missing some discussion forum where this matter is discussed?

**Comment by laurit** [comment, marked as answer]
A stable instrumentation requires stable semantic conventions, https://github.com/open-telemetry/semantic-conventions/blob/main/specification/trace/semantic_conventions/messaging.md needs to be stabilized first.

**Reply by asafch** [reply]
I see, thank you very much!
